opencensus/proto/trace/trace.proto

```diff
@@ -47,6 +47,23 @@ message Span {
   // This field is required.
   bytes span_id = 2;
 
+  // The `tracestate` field conveys information about request position in multiple distributed
+  // tracing graphs.
+  //
+  // There can be a maximum of 32 members in the map.
+  //
+  // The key must begin with a lowercase letter, and can only contain lowercase letters 'a'-'z',
+  // digits '0'-'9', underscores '_', dashes '-', asterisks '*', and forward slashes '/'. For
+  // multi-tenant vendors scenarios '@' sign can be used to prefix vendor name. The maximum length
+  // for the key is 256 characters.
+  //
+  // The value is opaque string up to 256 characters printable ASCII RFC0020 characters (i.e., the
+  // range 0x20 to 0x7E) except ',' and '='. Note that this also excludes tabs, newlines, carriage
+  // returns, etc.
+  //
+  // See the https://github.com/w3c/distributed-tracing for more details about this field.
+  map<string, string> tracestate = 15;
```

Comment:
nit: consider ordering the fields by its field number. Same for `kind`.

Reply:
Is `map` ordered in proto? Not that important for storing as for propagation, but nevertheless for consistency it may be better to have a list